chromium_src/chrome/installer/mini_installer/brave_mini_installer_unittest.cc

```diff
@@ -23,37 +23,54 @@ TEST_F(BraveMiniInstallerTest, HasNoReferralCode) {
 
 TEST_F(BraveMiniInstallerTest, HasStandardReferralCode) {
   ReferralCodeString referral_code;
-  EXPECT_TRUE(ParseReferralCode(L"BraveBrowserSetup-FOO123.exe", referral_code));
+  EXPECT_TRUE(ParseReferralCode(L"BraveBrowserSetup-FOO123.exe",
+                          referral_code));
+  EXPECT_STREQ(referral_code.get(), L"FOO123");
+}
+
+TEST_F(BraveMiniInstallerTest, HasStandardReferralCode) {
```

Comment:
Is this test name the same as the previous test name? Is that legal? Even if it is, it might get a little confusing.